lib/validate.js

```diff
@@ -41,6 +41,20 @@ async function validateMetadata(descriptor) {
 
   const profile = await Profile.load(defaultProfile)
 
+  // TODO: remove this temporal fix, when
+  // TODO: https://github.com/frictionlessdata/tableschema-js/issues/133 is resolved
+  // datahub.io remote site Fails if there is  '../'  in the resource.path
+  // https://github.com/datahq/data-cli/issues/265
+  descriptor.resources.map(res => {
+    if (res.path.includes('../') || res.path.startsWith('/')){
+      throw new Error(
+        res.path + '\n' +
+        "Absolute path (if starts from '/') and relative parent path (if includes '../')\n" +
```

Comment:
And error message like them would be enough I think https://github.com/frictionlessdata/datapackage-py/blob/master/datapackage/resource.py#L479